hindi/swami-vivekananda/swami-vivekananda.md

```diff
@@ -0,0 +1 @@
+﻿पता है स्वामी विवेकानंद जी क्या कहते थे? [यहाँ देखो|](swami-vivekananda/teachings-of-swami-vivekananda/quotes.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.